README.md

```diff
@@ -33,6 +33,85 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Getting started
+
+### Prerequisites
+`runc` is expected to be installed on the system. It is picked up as the default runtime by ocid.
+
+Build
+
+```
+$ GOPATH=/path/to/gopath
+$ mkdir $GOPATH
+$ go get -d github.com/kubernetes-incubator/cri-o
+$ cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
+$ make binaries
+$ sudo make install
```

Comment:
also since `command` is actually a shell-defined function, `GO_MD2MAN ?= $(shell command -v go-md2man')` doesnt work for me, but `GO_MD2MAN ?= $(shell /bin/sh -c 'command -v go-md2man')` works :(